src/estimators/same_key_aggregator.py

```diff
@@ -123,20 +127,30 @@ def __init__(self,
       epsilon_split: The proportion of total privacy budget that is assigned to
         the estimate of 1+ reach. The remaining privacy budget is assigned to
         the frequency histogram among effective keys.
+      reach_noiser_para: a dictionary of arguments other than epsilon in the
+        reach noiser. Example arguments are delta and random_state.
```

Comment:
Could you document this a little more carefully?  E.g., specify what the data types of delta and random_state are, how they are used, and what happens if they are omitted?

Reply:
Thank you Matthew. 

Motivated by your comments + your design in the planning_eval_fw, I am working on an another way to refactor the codes. Will hopefully cook it out tomorrow, push it to another branch, and let you review.